src/test/unit/actions/otpActions/otpActions.test.ts

```diff
@@ -0,0 +1,52 @@
+import { OtpActions } from '../../../../main/resources/actions/otpActions';
+
+const validOtpCode = 123456;
+const invalidOtpCode = 123457;
+const email = 'pippo@pluto.com';
+
+const otpActions = new OtpActions();
+
+describe(`validateOtp(${validOtpCode})`, () => {
+  const validateOtp = otpActions.validateOtp(validOtpCode, email);
+
+
+  it('should have return object with validOtp true', () => {
+    expect(validateOtp.otpValid).toBe(true);
+  });
+
+});
+
+describe(`validateOtp(${invalidOtpCode})`, function () {
+  const validateOtp = otpActions.validateOtp(invalidOtpCode, email);
+
+  it(`should return an object with validOtp false`, () => {
+    expect(validateOtp.otpValid).toBe(false);
+  });
+});
+
+describe(`getAttempts(${email})`, function () {
```

Comment:
you dont need describes for every test, this is over using descirbes, you only really need 2 describes in this file